scratch/nanotags-light-dom/tokens.css

```
@@ -0,0 +1,24 @@
+:root {
+  color-scheme: light;
+  --p-color-canvas: light-dark(#fff, hsl(225 66.7% 1.2%));
+  --p-color-surface: light-dark(hsl(240 10% 95%), hsl(240 2% 10%));
+  --p-color-primary: light-dark(hsl(225 66.7% 1.2%), hsl(225 100% 99%));
+  --p-color-error: light-dark(hsl(357 78% 41%), hsl(0 96.9% 62%));
+  --p-color-focus: light-dark(#1a44ea, #1a44ea);
+  --p-color-contrast-medium: light-dark(hsl(240 6.1% 7% / 0.6), hsl(240 12.5% 96.9% / 0.56));
+  --p-font-porsche-next: "Porsche Next", "Arial Narrow", Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)